src/components/Modal/BaseModal/BaseModal.tsx

```diff
@@ -1,12 +1,23 @@
-import { useEffect, useRef } from "react";
+import { useEffect, useRef, ReactNode } from "react";
 import styles from "./base.module.css";
 import closeIcon from "assets/images/ic_close.png";
+import { CloseModal } from "../../../utils/hooks/useModal";
 
-function BaseModeal({ title, children, variant, closeModal }) {
-  const modalRef = useRef(null);
+export interface ModalProps {
+  variant: string;
+  closeModal: CloseModal;
```

Comment:
이렇게 간단한 타입들은 인라인으로 작성하지않고 오히려 별도로 관리하면 이해하는데 코드를 이해하는데 어려울 수도 있는데요~ 과연 추상화가 적절하게 된 것인지에 대한 고민이 필요한 것 같아요.

아래와 같이 인라인으로 작성되는게 CloseModal을 클릭해서 다른 파일로 넘어가서 타입을 이해하는 것보다 적절하지 않을까요?
```suggestion
  closeModal: (modalName: string) => void;
```